client/src/components/TimetableSwitcher/TimetableSwitcher.tsx

```diff
@@ -28,7 +31,10 @@ const TimetableSwitcher = memo(() => {
 	const activeTimetableName = useSelector(
```

Comment:
This wasn't in your changes, but could use an update to use the exported selector from `selectors/timetable.ts`
```suggestion
	const activeTimetableName = useSelector(selectActiveTimetableName);
```